packages/shared/src/components/Feed.tsx

```diff
@@ -241,12 +242,6 @@ export default function Feed<T>({
     selectedPostIndex,
   } = usePostModalNavigation(items, fetchPage, updatePost, canFetchMore);
 
-  useEffect(() => {
```

Comment:
One less useEffect. Makes test easier as well.

Reply:
Was no longer used anywhere?
(missing context why it was needed/no longer needed)

Reply:
When we return an empty feed, edge case when users tags are too narrow and literally no content is shown.